libraries/support/azure/service.rb

```diff
@@ -69,21 +69,25 @@ def set_reader(name, value, override)
       self
     end
 
-    def get(url:, api_version:, error_handler: nil, unwrap: nil)
+    def get(use_cache = true, url:, api_version:, error_handler: nil, unwrap: nil)
```

Comment:
There's no need to add a positional argument for this, and I'd prefer you didn't.
```ruby
def get(url:, api_version:, error_handler: nil, unwrap: nil, use_cache: true)
```
```ruby
get(url: '/keys', api_version: '2016-10-01', use_cache: false)
```